### PR TITLE
4601 display dollar currency price

### PR DESCRIPTION
**Pull request by andrewkryshtal**

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -586,7 +586,7 @@ SPEC CHECKSUMS:
   CocoaAsyncSocket: 065fd1e645c7abab64f7a6a2007a48038fdc6a99
   DoubleConversion: cf9b38bf0b2d048436d9a82ad2abe1404f11e7de
   FBLazyVector: 49cbe4b43e445b06bf29199b6ad2057649e4c8f5
-  FBReactNativeSpec: 091e3ebb4e41b04653769e622c0dc90f8670318e
+  FBReactNativeSpec: cfcd14f56ff0e9e07e965d363a0fb0384dc6a5ec
```
Comment (keshan3262) [inline ios/Podfile.lock:589]: Is this change necessary? Sorry, I really don't know

File: src/hooks/use-data-load-timeout.hook.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+export const useInitDataLoadTimeout = (callback: () => void, refreshInterval: number) => {
+  const isAuthorised = useIsAuthorisedSelector();
+  const selectedAccount = useSelectedAccountSelector();
+
+  useEffect(() => {
+    callback();
+  }, []);
+
+  useEffect(() => {
+    callback();
```
Comment (keshan3262) [inline src/hooks/use-data-load-timeout.hook.ts:14]: I think there is no need in doing the same stuff twice

Comment (keshan3262) [reply src/hooks/use-data-load-timeout.hook.ts:14]: I mean that this hook https://github.com/madfish-solutions/templewallet-mobile/pull/135/files#diff-fd7d179656c1977613b419dbdde9a1646eea8a32f5909b39d15198dfa2845925R9 can be removed

File: ios/Podfile.lock (hunk 1 above)
Comment (andrewkryshtal) [reply ios/Podfile.lock:589]: not necessary, because every new build generates new podfile hashes

**Review by keshan3262 — APPROVED** [review]

File: ios/TempleWallet.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -542,7 +542,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
-				CURRENT_PROJECT_VERSION = 35;
+				CURRENT_PROJECT_VERSION = 34;
```
Comment (unixvb) [inline ios/TempleWallet.xcodeproj/project.pbxproj:545]: revert this

File: src/components/token-dropdown/token-dropdown-item/token-dropdown-item.tsx [hunk 4]
```diff
@@ -35,9 +40,14 @@ export const TokenDropdownItem: FC<Props> = ({ token = emptyToken, actionIconNam
...
       </View>
       <View style={styles.rightContainer}>
-        <Text style={styles.balance}>
-          {formattedBalance} {symbol}
-        </Text>
+        <View>
+          <Text style={styles.balance}>
+            {formattedBalance} {symbol}
+          </Text>
+          <Text style={styles.dollarEquivalent}>
```
Comment (unixvb) [inline src/components/token-dropdown/token-dropdown-item/token-dropdown-item.tsx:47]: why don't just pass this styles into `DollarEquivalentText` component?

File: src/hooks/use-data-load-timeout.hook.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
+
+import { useIsAuthorisedSelector, useSelectedAccountSelector } from '../store/wallet/wallet-selectors';
+
+export const useInitDataLoadTimeout = (callback: () => void, refreshInterval: number) => {
```
Comment (unixvb) [inline src/hooks/use-data-load-timeout.hook.ts:5]: use `EmptyFn` type

File: src/hooks/use-data-load-timeout.hook.ts [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    callback();
+
+    let timeoutId = setTimeout(function updateData() {
+      callback();
+      timeoutId = setTimeout(updateData, refreshInterval);
+    }, refreshInterval);
+
+    return () => {
+      clearTimeout(timeoutId);
+    };
```
Comment (unixvb) [inline src/hooks/use-data-load-timeout.hook.ts:19]: short return

File: src/hooks/use-data-load-timeout.hook.ts [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+  }, [isAuthorised, selectedAccount.publicKeyHash]);
```
Comment (unixvb) [inline src/hooks/use-data-load-timeout.hook.ts:20]: `isAuthorised` flag is not used :((

File: src/store/currency/currency-epics.ts [hunk 8]
```diff
@@ -14,28 +15,20 @@ export const loadExchangeRatesEpic = (action$: Observable<Action>) =>
...
+          ];
         }),
         catchError(err => of(loadExchangeRates.fail(err.message)))
       )
     )
   );
 
-export const loadTezosExchangeRateEpic = (action$: Observable<Action>) =>
-  action$.pipe(
-    ofType(loadTezosExchangeRate.submit),
```
Comment (unixvb) [inline src/store/currency/currency-epics.ts:32]: I think you don't need this action any more

File: src/hooks/use-data-load-timeout.hook.ts (hunk 7 above)
Comment (unixvb) [reply src/hooks/use-data-load-timeout.hook.ts:20]: wrong dependencies at all

File: src/hooks/use-authorized-timer-effect.hook.ts [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+      callback();
+
+      let timeoutId = setTimeout(function updateData() {
+        callback();
+        timeoutId = setTimeout(updateData, refreshInterval);
+      }, refreshInterval);
+
+      return () => clearTimeout(timeoutId);
+    }
+  }, [isAuthorised, ...[deps]]);
```
Comment (unixvb) [inline src/hooks/use-authorized-timer-effect.hook.ts:20]: `...[deps]` this's wrong.  `deps` is an array already

**Review by unixvb — APPROVED** [review]
